src/services/MemberTraitService.js

```diff
@@ -206,13 +206,6 @@ async function updateTraits (currentUser, handle, data) {
     }
   })
 
-  if(await lookerService.isMemberVerified(member.userId)){
```

Comment:
Why do we need to remove this?

Reply:
We don't need to do this in the TraitService - only the member service.  This was just leftover test code that I failed to remove that's causing a problem in dev